pkg/gofr/container/container.go

```diff
@@ -61,6 +61,7 @@ type Container struct {
 	OpenTSDB   OpenTSDB
 	ScyllaDB   ScyllaDB
 	SurrealDB  SurrealDB
+	Arango     ArangoDB
```

Comment:
Please respect the name

```suggestion
	ArangoDB     ArangoDB
```

Reply:
sure done